README.md

```diff
@@ -26,7 +26,8 @@ use it with:
 
 ## Example usage
 
-We don't offer a binary distribution, so first you need to build from source:
+We don't offer a binary distribution, so first you need to build from source.
+From top level run:
```

Comment:
"top level" is a bit ambiguous, how about "from the root directory"

Reply:
Made the change you suggested.